packages/@aws-cdk/toolkit/lib/toolkit/toolkit.ts

```diff
@@ -785,6 +785,8 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
     return new Deployments({
       sdkProvider: await this.sdkProvider(action),
       toolkitStackName: this.toolkitStackName,
+      ioHost: this.ioHost as any, // @todo temporary while we have to separate IIoHost interfaces
```

Comment:
why _do_ we have separate `IIoHost` interfaces?

Reply:
Because currently the toolkit depends on the CLI package. Once the repo split is complete, we will introduce a new private intermediate package that both toolkit and CLI can depend on.